### PR TITLE
NAS-127472 / 24.10 / Refactoring page headers

### PR DESCRIPTION
**Pull request by undsoft**
Previous implementation didn't allow tests to interact with button in page headers. This one does.

Testing: check page headers for regressions.

### CODE REVIEW

**Review by denysbutenko — APPROVED** [review]
The Groups page is missing the `Priveledges` button for some reason. Then I switched to the `Directory Services`, and there was no page header.

Overall, it looks nice.

File: src/app/pages/system/advanced/cron/cron-list/cron-list.component.html [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 
-    <ix-table-columns-selector [columns]="columns" (columnsChange)="columnsChange($event)"></ix-table-columns-selector>
+  <ix-table-columns-selector [columns]="columns" (columnsChange)="columnsChange($event)"></ix-table-columns-selector>
 
-    <button mat-button color="primary" id="lol" ixTest="add-cronjob" (click)="doAdd()">
-      {{ 'Add' | translate }}
-    </button>
-  </ix-page-title-header>
-</ng-template>
+  <button mat-button color="primary" id="lol" ixTest="add-cronjob" (click)="doAdd()">
```
[the remark this answers is not in the document]
Comment (undsoft) [reply src/app/pages/system/advanced/cron/cron-list/cron-list.component.html:6]: lol indeed

**Comment by undsoft** [comment]
I fixed the Privileges button, but I cannot reproduce the issue with Directory Services.

**Comment by denysbutenko** [comment]
> I fixed the Privileges button, but I cannot reproduce the issue with Directory Services.

I'll double-check when a fresh image is built.

**Comment by denysbutenko** [comment]
Regression on the Directory Services page persists.
image

image

**Pull request merged**